.editorconfig

```diff
@@ -10,6 +10,7 @@ tab_width = 4
 
 # ktlint
 ij_kotlin_imports_layout = *,java.**,javax.**,kotlin.**,^
+ij_kotlin_allow_trailing_comma = true
```

Comment:
Weird. Setting this for me locally caused ktlint failures when running from command line...

Reply:
I would have expected that you'd also see failures. And I think I see why: 

> Task :file-attribute-caching:ktlintMainSourceSetCheck FROM-CACHE

Reply:
For future reference, it was actually because of the need for `ij_kotlin_allow_trailing_comma_on_call_site = true`